lte/gateway/c/core/oai/tasks/amf/amf_smf_send.cpp

```diff
@@ -824,4 +885,182 @@ int amf_max_pdu_session_reject(
   }
   return rc;
 }
+
+/****************************************************************************
+ **                                                                        **
+ ** Name        :  construct_pdu_session_reject_dl_req()                   **
+ **                                                                        **
+ ** Description :  Construct Session Establishment Reject Struct           **
+ **                                                                        **
+ ** Inputs      :  sequence_number     : seq to construct Secure msgs      **
+ **                session_id	       : PDU Session Identity              **
+ **   		   pti                 : Procedure transaction identity    **
+ **		   is_security_enabled : indcates to construct plain msg   **
+ **				         secure msg                        **
+ **   		   msg                 : out parmaeter Session             **
+ **                                      Establishment Reject              **
+ **                                                                        **
+ **  Return     :  len                 : buffer required for               **
+ **                                      DLNASTransportMsg                 **
+ **                                                                        **
+ ***************************************************************************/
+
+int construct_pdu_session_reject_dl_req(
```

Comment:
Mostly it seems will be used here. Can you please make it static. Also in test program call the parent function. Check the possibility.